specifyweb/frontend/js_src/lib/components/SchemaConfig/Fields.tsx

```diff
@@ -91,6 +81,23 @@ export function SchemaConfigFields({
   );
 }
 
+export function SchemaConfigFieldsList({
+  fields,
+}: {
+  fields: readonly SerializedResource<SpLocaleContainerItem>[];
```

Comment:
for consistency, consider using `RA<T>` instead of `readonly T[]` syntax

a blurb from Google Bard on consistency:

> Consistency in code is important because it leads to predictability. Consistency makes code more readable and understandable, which is important for both the developer who wrote the code and any other developers who may need to work with it in the future. Consistency in naming conventions, indentation, and formatting can also make it easier to spot errors and bugs in the code.
>
> Consistent coding standards also facilitate seamless collaboration among team members, improving productivity and reducing conflicts. When code is consistent, maintaining and updating it becomes easier, even for developers who weren't involved in the initial development.
>
>A consistent codebase is easier to work with because you get used to its patterns. Your brain creates shortcuts that help you read its code faster.